plugins/grpc/kotlin/v1.3.0/Dockerfile

```diff
@@ -8,11 +8,7 @@ RUN apt-get update \
  && apt-get install -y curl
 RUN curl -fsSL -o protoc-gen-grpc-kotlin.jar https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-kotlin/1.3.0/protoc-gen-grpc-kotlin-1.3.0-jdk8.jar
 
-FROM eclipse-temurin:17.0.4_8-jre-jammy
+FROM gcr.io/distroless/java11-debian11
```

Comment:
Java 11 distroless image is ~21MB smaller than the Java 17 one.

Reply:
Are we sure we want to tradeoff an older (LTS) for a newer one based on the image size?

Reply:
It felt like a good trade off (at least until it is no longer supported).